.gitignore

```diff
@@ -1,9 +1,7 @@
 .DS_Store
 
-build/
 platforms/
 plugins/
-www/
```

Comment:
I added a .gitignore file in www, because cordova requires that the www directory exists, but I don't want to actually version control anything in there